doc/content/source/equation_systems/equation_system.md

```diff
@@ -0,0 +1,50 @@
+# EquationSystem
+
+The EquationSystem is responsible for defining and assembling the weak form of the PDE into an
+`mfem::Operator` used to solve an iteration of the FE problem. This operator is passed to an
```

Comment:
Consider adding links to the relevant MFEM docs.

Reply:
Added